alert-policies/lampy/CpuUtilization.yml

```diff
@@ -0,0 +1,35 @@
+name: High CPU Utilization
+
+description: |+
+  This alert is triggered when the CPU Utilization is above 90% for 5 minutes.
```

Comment:
before: `This alert is triggered when the CPU Utilization is above 90% for 5 minutes.`
can be changed to: `This alert is triggered when the CPU utilization exceeds 90% for 5 minutes.`